package.json

```diff
@@ -83,6 +84,7 @@
 		"tree-sitter-java": "^0.19.1",
 		"tree-sitter-javascript": "^0.19.0",
 		"tree-sitter-json": "^0.19.0",
+		"tree-sitter-latex": "github:latex-lsp/tree-sitter-latex#1666e9780de42a31b1376d32fbe8332e8dd5850f",
```

Comment:
@wenkokke as newly crowned dependency-management czar for this repo, does this line meet your approval? 😊

Reply:
Looks great! Very happy for latex support!